### PR TITLE
i18n(fr): Update `guides/backend/google-firebase.mdx` from #8014 (__session)

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Update `guides/backend/google-firebase.mdx` from #8014 (__session) + removing space encoding

#### Related issues & labels (optional)

- Closes #
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/backend/google-firebase.mdx [hunk 1]
```diff
@@ -20,13 +20,13 @@ Consultez notre guide dédié au [déploiement sur Firebase hosting](/fr/guides/
...
+- Des identifiants Firebase : vous aurez besoin de deux ensembles d'identifiants pour connecter Astro à Firebase :
+  - Les identifiants de l'application Web : ces informations d'identification seront utilisées par le côté client de votre application. Vous pouvez les trouver dans la console Firebase sous *Paramètres du projet > Général*. Faites défiler jusqu'à la section **Vos applications** et cliquez sur l'icône **Application Web**.
+  - Les informations d'identification du projet : ces informations d'identification seront utilisées par le côté serveur de votre application. Vous pouvez les générer dans la console Firebase sous *Paramètres du projet > Comptes de service > Firebase Admin SDK > Générer une nouvelle clé privée*.
 
 ### Ajouter les informations d'identification Firebase
 
-Pour ajouter vos informations d'identification Firebase à Astro, créez un fichier « .env » à la racine de votre projet avec les variables suivantes :
+Pour ajouter vos informations d'identification Firebase à Astro, créez un fichier « .env » à la racine de votre projet avec les variables suivantes :
```
Comment (Fryuni) [inline src/content/docs/fr/guides/backend/google-firebase.mdx:29]: What are all these whitespace changes? Maybe there is some encoding issue here

File: src/content/docs/fr/guides/backend/google-firebase.mdx [hunk 2]
```diff
@@ -219,14 +219,18 @@ export const GET: APIRoute = async ({ request, cookies, redirect }) => {
...
+  cookies.set("__session", sessionCookie, {
     path: "/",
   });
 
   return redirect("/dashboard");
 };
 ```
 
+:::caution
+Firebase n'autorise que l'utilisation d'un seul cookie, qui doit être nommé `__session`](https://firebase.google.com/docs/hosting/manage-cache#using_cookies). Tout autre cookie envoyé par le client ne sera pas visible par votre application.
```
Comment (Fryuni) [inline src/content/docs/fr/guides/backend/google-firebase.mdx:231]: It is missing the start of the link
```suggestion
Firebase n'autorise que l'utilisation [d'un seul cookie, qui doit être nommé `__session`](https://firebase.google.com/docs/hosting/manage-cache#using_cookies). Tout autre cookie envoyé par le client ne sera pas visible par votre application.
```

**Review by Fryuni — CHANGES_REQUESTED** [review]
Thank you for the PR!

There seems to be some problem with whitespace and a broken link syntax.

File: src/content/docs/fr/guides/backend/google-firebase.mdx (hunk 1 above)
Comment (thomasbnt) [reply src/content/docs/fr/guides/backend/google-firebase.mdx:29]: Hello! Yeah this is the encoding, I removed that

File: src/content/docs/fr/guides/backend/google-firebase.mdx (hunk 2 above)
Comment (thomasbnt) [reply src/content/docs/fr/guides/backend/google-firebase.mdx:231]: Oh thanks!

**Review by dreyfus92 — APPROVED** [review]
Merci @thomasbnt, LGTM ✅

**Review by yanthomasdev — APPROVED** [review]
LGTM!

**Review by Fryuni — APPROVED** [review]

**Comment by McFlyPartages** [comment]
LGTM

**Review by McFlyPartages — APPROVED** [review]

**Pull request merged**